### ISSUE TITLE
[bug] free server slot display error

### ISSUE THREAD

**Issue by Disastur** [Minehut/Issues#244]
### Expected behavior
default tier server displays #/4 in the server list at all times 

### Actual behavior

![image](https://user-images.githubusercontent.com/29286256/28645234-0e9d7594-725d-11e7-96cd-0b96ea4e477b.png)

### Detailed steps to reproduce this behavior
look into the server list, some are glitched

### Server name
This server is First, others appear to be glitched too

**Comment by Disastur** [comment]
(this screenshot was taken on July 26th 2017)

**Comment by iGalaxyYT** [comment]
This is correct, no bug, old free servers just had 10 player slots and it doesnt get removed until they upgrade or reset all.

**Comment by Disastur** [comment]
@iGalaxyYT no, old free servers kept their plugins and world border, but the plugin slots got reduced to two and the player slots to four.

**Comment by iGalaxyYT** [comment]
No, old free serverd that hadn't reset yet kept their plugin and player slots @Disastur https://minehut.com/forums/topics/595db2525844a3514ff86f84
